maintest.txt

```diff
@@ -1 +1,2 @@
 Jesus is Lord and for God so love the world that he gave his only begotten son that whosoever believe in him should not perish but have evalasting live. 
```

Comment:
This is great but fix typo with evalasting

Reply:
I have fixed the typo with everlasting